staging/src/k8s.io/kms/go.mod

```diff
@@ -19,11 +20,14 @@ require (
 	golang.org/x/net v0.7.0 // indirect
 	golang.org/x/sys v0.5.0 // indirect
 	golang.org/x/text v0.7.0 // indirect
+	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8 // indirect
 	google.golang.org/genproto v0.0.0-20220502173005-c8bf987b8c21 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
 )
 
 replace (
+	k8s.io/api => ../api
```

Comment:
Why is `k8s.io/api` required? I think this can be removed. If this is removed, you can also remove `api` from the publishing rules.

Reply:
I _think_ `client-go` enforces an indirect dependency on `api`.

Reply:
I just cloned this PR and tried removing it. This replace directive is not required.

Reply:
AFAIK the replace directive is always required if you have the dependancy at all, otherwise you will pull in the wrong version.